http3/client.go

```diff
@@ -151,11 +155,14 @@ func (c *client) RoundTrip(req *http.Request) (*http.Response, error) {
 		return nil, c.handshakeErr
 	}
 
-	// wait for the handshake to complete
-	select {
-	case <-c.session.HandshakeComplete().Done():
-	case <-req.Context().Done():
-		return nil, req.Context().Err()
+	// Immediately send out this request, if this is a 0-RTT request.
+	if req.Method != MethodGet0RTT {
```

Comment:
Do we want to rewrite the method name here, so that it doesn't get sent to the server like this?

Reply:
Good catch. Forgot that.